utils/dockerutil.py

```diff
@@ -232,6 +232,17 @@ def get_hostname(self, use_default_gw=True, should_resolve=False):
 
         return self._default_gateway
 
+    def get_host_tags(self):
+        tags = []
+        version = self.client.version()
+        if version and 'Version' in version:
+            tags.append('docker_version:%s' % version['Version'])
+        else:
+            log.debug("Could not determine Docker version")
+        if self.is_swarm():
+            tags.append('docker_swarm:active')
```

Comment:
Maybe not for the user, but for support and usage statistics I think that's relevant